shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/Literals.g4

```diff
@@ -20,7 +20,7 @@ lexer grammar Literals;
 import Alphabet, Symbol;
 
 IDENTIFIER_
-    : [A-Za-z_$0-9]*?[A-Za-z_$]+?[A-Za-z_$0-9]*
+    : [A-Za-z]+[A-Za-z_$#0-9]*
```

Comment:
The 'IDENTIFIER' should not be modified. It's better to identify a new lexer rule to be used by password.

Reply:
Hi @wgy8283335, I've tried to create a new lexer rule for `password`, however `password` has really similar pattern (for example `abc123`) as the previous `IDENTIFIER_` so that most of them will be first assigned as `IDENTIFIER_`. So I think we can't remove `IDENTIFIER_` from password, otherwise we will have `no viable alternative at input 'abc123'` error. 

Hier is the [doc](https://github.com/antlr/antlr4/blob/a60c32d36933758b63662cdade0a2705ca460804/doc/wildcard.md#nongreedy-lexer-subrules) I'm referencing:

> If more than one lexer rule matches the same input sequence, the priority goes to the rule occurring first in the grammar file.

Do you have any suggestions how to overcome this issue?

Reply:
@Liangda-w. Hi, may like this:
```
PASSWORD_: [A-Za-z]+[A-Za-z_$#0-9]*;
IDENTIFIER_:[A-Za-z_$0-9]*?[A-Za-z_$]+?[A-Za-z_$0-9]*;
```
Keep `IDEANTIFIER` not be changed. Add a rule `PASSWORD_`. 
The 'password ' rule only use PASSWORD.
```
password: PASSWORD_;
```